site/class-3-clean/vincentkempers/index.html

```diff
@@ -0,0 +1,229 @@
+
+path, line {
+  stroke: currentcolor;
+}
+
+text {
+  font-size: small;
+  fill: currentcolor;
+}
+</style>
+<svg width=960 height=500 viewbox="0 0 960 500"></svg>
+<script src=https://d3js.org/d3.v4.min.js></script>
+<script>
+var svg = d3.select('svg');
+var margin = {top: 20, right: 140, bottom: 30, left: 50};
+var width = svg.attr('width') - margin.left - margin.right;
+var height = svg.attr('height') - margin.top - margin.bottom;
+var g = svg.append('g').attr('transform', 'translate(' + margin.left + ',' + margin.top + ')');
+
+var x = d3.scaleTime().range([0, width]);
+var y = d3.scaleLinear().range([height, 0]);
+var z = d3.scaleOrdinal(['#feca2f', '#2ffe63', '#2f63fe', '#fe2fca']);
+
+var knmiCodes = {
+  209: 'IJmond',
+  210: 'Valkenburg',
+  215: 'Voorschoten',
+  225: 'IJmuiden',
+  235: 'De Kooy',
+  240: 'Schiphol',
+  242: 'Vlieland',
+  248: 'Wijdenes',
+  249: 'Berkhout',
+  251: 'Hoorn (Terschelling)',
+  257: 'Wijk aan Zee',
+  258: 'Houtribdijk',
+  260: 'De Bilt',
+  265: 'Soesterberg',
+  267: 'Stavoren',
+  269: 'Lelystad',
+  270: 'Leeuwarden',
+  273: 'Marknesse',
+  275: 'Deelen',
+  277: 'Lauwersoog',
+  278: 'Heino',
+  279: 'Hoogeveen',
+  280: 'Eelde',
+  283: 'Hupsel',
+  285: 'Huibertgat',
+  286: 'Nieuw Beerta',
+  290: 'Twenthe',
+  308: 'Cadzand',
+  310: 'Vlissingen',
+  311: 'Hoofdplaat',
+  312: 'Oosterschelde',
+  313: 'Vlakte v.d. Raan',
+  315: 'Hansweert',
+  316: 'Schaar',
+  319: 'Westdorpe',
+  323: 'Wilhelminadorp',
+  324: 'Stavenisse',
+  330: 'Hoek van Holland',
+  331: 'Tholen',
+  340: 'Woensdrecht',
+  343: 'R’dam-Geulhaven',
+  344: 'Rotterdam',
+  348: 'Cabauw',
+  350: 'Gilze-Rijen',
+  356: 'Herwijnen',
+  370: 'Eindhoven',
+  375: 'Volkel',
+  377: 'Ell',
+  380: 'Maastricht',
+  391: 'Arcen'
+};
+
+// You may start editing stuff here (but don’t change the file-name).
+d3.text('index.txt')
+  .get(onload);
+
+function onload (err, doc) {
+  if (err) throw err;
+
+  // Tijd moet isco format hebben "2017-08-31T23:00:00.000Z"
+
+  // Use this place to transform doc (without changing it manually) into
+  // something that looks like `example.json`:
+  // https://github.com/cmda-fe3/course-17-18/blob/master/site/class-3/clean/example.json
+  //
+  // Notes:
+  //
+  // 1. Use the `knmiCodes` variable to transform from KNMI’s codes to place
+  //    names
+  // 2. Use actual Date objects instead of strings for all `date` keys:
+  //    https://developer.mozilla.org/JavaScript/Reference/Global_Objects/Date
+  //
+  // Store the results in the variable `places`:
+  // variabelen
+
+  var deleteHeader = doc.indexOf('STN,YYYYMMDD');
+  doc = doc.slice(deleteHeader);
+  var end = doc.indexOf('\n', deleteHeader);
+  doc = doc.slice(end).trim();
+  doc = doc.replace(/ +/g, '');
+  var parseTime = d3.utcParse('%Y%m%dT%H:%M:%S.%LZ'),
+    places = d3.csvParseRows(doc, map);
+
+  function map (d) {
+    return {
+      name: knmiCodes[d[0]],
+      date: parseTime(d[1].concat('T', d[2], ':00:00.000Z')),
```

Comment:
What if you use a different format on line 133? Then you don’t need to add this `:00:00.000z` part?